packages/esm-messages-app/src/messages/messages-chart-dashboard.component.tsx

```diff
@@ -0,0 +1,195 @@
+import React, { useCallback, useMemo } from 'react';
+import classNames from 'classnames';
+import { useTranslation } from 'react-i18next';
+import {
+  Button,
+  DataTable,
+  DataTableSkeleton,
+  InlineLoading,
+  Layer,
+  OverflowMenu,
+  OverflowMenuItem,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableHeader,
+  TableRow,
+  Tile,
+} from '@carbon/react';
+import { Add } from '@carbon/react/icons';
+import { useConfig, useLayoutType } from '@openmrs/esm-framework';
+import { CardHeader, EmptyState, ErrorState, launchPatientWorkspace } from '@openmrs/esm-patient-common-lib';
+import { useMessagesTemplates } from '../hooks/useMessagesTemplates';
+import styles from './messages-chart-dashboard.scss';
+import { type MessagesTemplate } from '../types';
+import { type ConfigObject } from '../config-schema';
+
+function MessagesChartDashboard() {
+  const { t } = useTranslation();
+  const displayText = t('messages', 'Messages');
+  const headerTitle = t('messages', 'Messages');
```

Comment:
I get that `displayText` and `headerTitle` are serving different roles, but it seems like we don't need two variables if they're always the same string?